packages/block-editor/src/components/block-mover/README.md

```diff
@@ -0,0 +1,35 @@
+# Block mover
+
+Block movers allow moving blocks inside the editor using up and down buttons.
+
+![Block mover screenshot](https://make.wordpress.org/core/files/2020/08/block-mover-screenshot.png)
+
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+
+## Development guidelines
+
+### Usage
+
+Shows the block mover buttons in the block toolbar.
+
+```jsx
+import { BlockMover } from '@wordpress/block-editor';
+<BlockMover clientIds={ [ clientId ] } />
```

Comment:
Can we do the same as BlockBreadcrumb here (wrapping?)

Reply:
Looks fine I think.